.github/workflows/build.yml

```diff
@@ -294,3 +255,27 @@ jobs:
            allowUpdates: true
            prerelease: true
            draft: true
+
+
+   release-devtools:
+     runs-on: ubuntu-latest
+     steps:
+       - uses: actions/checkout@v2
+       - uses: ./.github/custom-actions/clojure-env
+       - uses: ./.github/custom-actions/node-env
+
+       - name: Build static Storybook site
+         run: yarn storybook:static
+
+       - name: Build static Clerk site
+         run: yarn notebooks:static
+
+       - uses: amondnet/vercel-action@v20
+         with:
+           vercel-token: ${{ secrets.VERCEL_TOKEN }}
+           vercel-org-id: ${{ secrets.VERCEL_ORG_ID }}
+           vercel-project-id: ${{ secrets.VERCEL_PROJECT_ID }}
```

Comment:
@tangjeff0 need you to set these three secrets. Am around to pair if you want.

The first one you can get from https://vercel.com/account/tokens.

The two other ones you can get from https://github.com/marketplace/actions/vercel-action#project-linking. You don't need to keep the `.vercel` folder around after getting them.